data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+			`	Several minutes later, you are escorted back to the base of the tower, which is now surrounded by a large group of Gegno civilians, all gawking at the two of you. You're led back to your ship with your head spinning slightly, unsure what to make of that whole encounter. As they did the first time you were with them on the one desert planet, the Gegno heads for your ship and hands you a small compass device, this time marking a different location on the map.`
+			`	At the very least, you can take this as an indication that they are not done with you yet, so you prepare to travel to your next destination.`
+				launch
+
+
+			label "done with the gegno"
+			`	For one last time, the trio beckons to the crowd followed by a long speech. Afterwards, the crowd goes silent. Your Gegno guide bumps into you and starts to direct you back to the cage lift with nobody else batting an eye. Several minutes later, you are escorted back to the base of the tower, still as barren as it was earlier. You're lead back to your ship with your head spinning slightly, unsure what to make of that whole encounter. The Gegno remains at a distance from your ship as you prepare to depart.`
+			`	Before you leave, you watch as they lift their staff weapon and flip it around. An explosive cloud of smoke follows, and a sizable crater appears just barely avoiding your ship on the ground. It seems that you're no longer welcome here, though, the fact that they intentionally missed such a large target might have other implications. Regardless, it could have been you on the other end of that blast at any point during this journey.`
+			`	Despite the aggressive action, you are not followed as you leave the planet.`
+				flee
+
+
+			label "no more gegno"
+			`	You ultimately decide that pursuing this ruse with the Gegno is no longer worth the effort. Even if you did meet up with them, or ended up in some sort of debate with them, it would likely take an excessive amount of effort in order to get any amount of progress, if it's even possible. Perhaps even landing here was a mistake, as you could never guess what might happen if you continued this endeavor, given you are all alone on the home world of a species that has shown itself to be rather xenophobic, even to the Quarg.`
+			`	As you prepare to depart, the Gegno, who was watching you all this time with a blank stare, lifts their staff weapon and flips it around. An explosive cloud of smoke follows, and a sizable crater appears just barely avoiding your ship on the ground. It seems they are, at the very least, unhappy with your decision to leave, which is a little odd considering they didn't want anything to do with you before. Though, the fact they intentionally missed such a large target might have other implications. Regardless, it could have been you on the other end of that blast.`
```

Comment:
As above.

Reply:
```suggestion
			`	As you prepare to depart, the Gegno, who was watching you all this time with a blank stare, lifts their staff weapon and flips it around. An explosive cloud of smoke follows, and a sizable crater appears right beside the <ship>. It seems they are, at the very least, unhappy with your decision to leave, which is a little odd considering they didn't want anything to do with you before. The fact that they intentionally missed such a large target, however, might have other implications. Regardless, it could have been you on the other end of that blast.`
```